qt/python/mantidqt/widgets/samplelogs/model.py

```diff
@@ -151,9 +184,21 @@ def getItemModel(self):
         onto a QTableView
         """
 
-        def create_table_item(column, itemname, callable, *args):
+        def create_table_item(column, itemname, invalid_value_count, log_size, callable, *args):
             item = QStandardItem()
             item.setEditable(False)
+            #format if there is invalid data entries
+            if invalid_value_count == -1:
+                item.setData(QColor.fromHsv(0, 180, 255), Qt.BackgroundRole)
```

Comment:
Perhaps consider pulling out `QColor.fromHsv(0, 180, 255)` as a named constant and the top of the module? It makes it a little clearer what is intended.